src/ppx_deriving.mli

```diff
@@ -72,35 +72,42 @@ val string_of_core_type : Parsetree.core_type -> string
     The [~name] argument is used in error messages and should receive
     the name of the deriving plugin, e.g. ["show"]. *)
 module Arg : sig
+  (** A type of conversion functions.
+
+      A conversion function of type ['a conv] converts a raw expression into an
+      argument of type ['a]. Returns [Result.Error "error"] if conversion fails
+  *)
```

Comment:
Much better but there's a dot missing at the end, also `or returns` sounds nicer.